webui.py

```diff
@@ -1275,12 +1306,15 @@ def packages_rss(self):
         if self.cache_redis is None:
             content = EMPTY_RSS
         else:
-            value = self.cache_redis.get('rss~pkgs')
-            if value:
-                content = value
-            else:
-                tasks.rss_regen()
-                content = self.cache_redis.get('rss~pkgs')
+            try:
+                value = self.cache_redis.get('rss~pkgs')
+                if value:
+                    content = value
+                else:
+                    tasks.rss_regen()
```

Comment:
since the shared RSS content cache currently handles regeneration on needed modifications for all nodes, it's not clear that serving three likely inconsistent sets of rss data is better than just one empty one until redis recovers. i'm gonna go ahead as this is.